pkgs/applications/science/math/wolfram-engine/l10ns.nix

```diff
@@ -7,6 +7,13 @@
 let allVersions = with lib; flip map
   # N.B. Versions in this list should be ordered from newest to oldest.
   [
+    {
+      version = "13.1.0";
+      lang = "en";
+      language = "English";
+      sha256 = "1659kyp38a8xknic95pynx9fsgn96i8jn9lnk89pc8n6vydw1460";
+      installer = "WolframEngine_13.1.0_LINUX.sh";
+    }
     {
       version = "13.0.1";
       lang = "en";
```

Comment:
Can we drop the other version(s)?

Reply:
No, if you have a commercial license for an older version, you still want to be able to use it. (The free version puts some usage limitations.)

Reply:
So are we collecting all versions going forward or what is the plan? Usually we try to keep as few versions possible.

Reply:
For  `mathematica` (same company) we have many versions from recent 13.0.1 back to 9.0.0 from 2012. I would do the same for the `wolfram-engine`.

Note that they are never built by hydra because you need to put the installer in the store manually (see my comment below). Only the persons who are interested in it will ever have to build its derivation.

Reply:
We should clean that up a bit to be honest.